actionpack/lib/action_controller/metal/etag_with_template_digest.rb

```diff
@@ -44,7 +44,7 @@ def determine_template_etag(options)
       # template digest from the ETag.
       def pick_template_for_etag(options)
         unless options[:template] == false
-          options[:template] || "#{controller_path}/#{action_name}"
+          options[:template] || lookup_context.find_all(action_name, _prefixes).first&.virtual_path
```

Comment:
`find` will raise an error if it doesn't find the template. I don't think we want to raise here since we didn't raise before. It fails a lot of actionpack tests that don't have templates but ask about the `etag`.

Maybe it would be worth switching to `find`? should we raise if it can't find the template you want as part of your etag? That would be new behavior, but might help people find problems with their cache keys? I am unsure.

Reply:
@HParker I would avoid trying to solve problems people aren't experiencing, this PR already adds enough value IMO ❤️